client/lib/analytics/super-props.js

```diff
@@ -29,6 +31,7 @@ const getSuperProps = ( reduxStore ) => ( eventProperties ) => {
 
 			site_id_label: selectedSite.jetpack ? 'jetpack' : 'wpcom',
 			site_plan_id: selectedSite.plan ? selectedSite.plan.product_id : null,
+			site: siteSlug,
```

Comment:
Is our goal to track `site` just for the domain only case? If so, could we have made the changes in `PageViewTracker` ? I'm hesitant to accept that change on this level because the super props are added to basically every analytics event we track. WDYT?

Reply:
Thanks for flagging this! It's certainly a lot more convenient to be able to set it once and for all in `super-props.js`, especially when some tracking calls aren't made from `<PageViewTracker>` but from calling `recordPageView()` where custom properties are passed. In the latter case, updates in `<PageViewTracker>` will not take effect. That said, I do agree that it's best to not blindly add this property to all track events, since the property may not be set up for all events.